src/server/game/Combat/CombatManager.cpp

```diff
@@ -0,0 +1,392 @@
+/*
+ * Copyright (C) 2008-2018 TrinityCore <http://www.trinitycore.org/>
```

Comment:
license is wrong

Reply:
not that i care too much but its not possible

Reply:
files created inside ac need to follow the same license, we cant have mixed licenses

Reply:
You cannot change a license simply because you wish to. That's not how copyright works. You can accrue licenses if different parts of a file have different licenses, as long as those licenses are compatible. But you are not the copyright holder of the work you're integrating. If as you say "license is wrong" then you simply cannot use the other programmers' work.

Reply:
We cannot merge this with the Trinity Core License still there, simple as that. So we would have to find some other way

Reply:
actually you cannot merge this in any other way than keeping that license as is

on top of that even though some months ago there was attempt to resolve AC license issue it was still done in wrong way and it is still not resolved and AC license is still invalid. from that point of view it doesnt even matter how this would be resolved

also there was discussion like 2 weeks ago about this in modding discord with shin and yehonal and they seemed to kind of understand that it is still wrong although im not sure if they know or want to fix it since they seem to stick with original idea of forcing people to share changes to core which is understandable but impossible and the only way to have valid license is to revert all files which originate from TC to TC license and then you are free to have whatever license in AC original files (not that its sensible because having different licenses in single project has complications but at least that would be possible way)

update in AC readme kind of suggests this 
- The new AzerothCore source components are released under the [GNU AGPL v3](https://www.gnu.org/licenses/agpl-3.0.en.html)
- The old sources based on MaNGOS/TrinityCore are released under the [GNU GPL v2](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html)
but in practice it is not respected and all files still contain wrong AC license

there was argument made by yehonal and shin that now when it is licensed like this they cant revert it to correct license because code contributed to AC since forking is licensed under AC license but that doesnt make any sense since that argument can be rephrased as "fixing invalid license is not possible because invalid license bounds us to respect it" which when worded like this is obvious oxymoron since you cant be bound to anything by invalid license

it could be argued in original AC files but that would be discussion for real lawyer and i have no idea if invalid license on project as whole can be valid on original files

Reply:
I'm not diving into that discussion as I was not part of it and I don't know what either Shin or Yehonal said, what I am saying is, as per FSF consultation, all files under AC need to follow the same license, as the license grants the software is distributed as a whole

This doesn't include deps ofc but it includes all files, so if this can't be changed, unfortunately there's no further point working this PR as it can't be merged like this. I am sorry